openandroidinstaller/assets/configs/Mi439.yaml

```diff
@@ -53,7 +53,7 @@ steps:
       content: >
         Install additional partitions selected before by pressing 'Confirm and run'. Once it's done continue.
         
-        Note : If you have not selected this partition, it will do nothing.
+        Note : If you have not selected additional partitions it will do nothing and is save to continue.
```

Comment:
This sentence sounds weird... Maybe it's better like this?: "[...] and it is save [...]"

Reply:
This is completely removed now, since this step is no longer required.